legion/engine/physics/physx/data/physx_wrapper_container.cpp

```diff
@@ -25,12 +25,12 @@ namespace legion::physics
         }
     }
 
-    std::optional<std::reference_wrapper< PhysxInternalWrapper>> PhysxWrapperContainer::findWrapperWithID(size_type id)
+    std::optional<core::pointer< PhysxInternalWrapper>> PhysxWrapperContainer::findWrapperWithID(size_type id)
```

Comment:
well the point of making it a pointer is also that it doesn't need to be optional anymore XD

Reply:
yeah fair XD